### PR TITLE
fix: Update of outdated data sets to 2019

### LINKED ISSUE #94: Use BAN instead of BD-TOPO
Instead of BD-TOPO for addresses, make use of the Base Adresse Nationale (BAN)
https://www.data.gouv.fr/fr/datasets/base-adresse-nationale/

### LINKED ISSUE #112: Update data to 2019
Census data is available for 2018, it should be updated.

### LINKED ISSUE #151: Missing files when gathering data to create the Île-de-France population
Hi @sebhoerl , When I followed the instruction to create population. 

In `Gathering the data` step 10: [Address database (BD-TOPO)](https://github.com/eqasim-org/ile-de-france/blob/develop/docs/population.md#10-address-database-bd-topo), I think the data from the provided link has b
een updated to a new version(December 2022 version).  Screenshot 2023-02-03 at 15 07 33

After I download the above dataset from this [link](https://wxs.ign.fr/859x8t863h6a09o9o6fy4v60/telechargement/prepackage/BDTOPOV3-TOUSTHEMES-REGION-PACK_224$BDTOPO_3-3_TOUSTHEMES_SHP_LAMB93_R11_2022-12-15/file/BDTOPO_3-3_TOUSTHEMES_SHP_LAMB93_R11_2022-12-15.7z), it doesn't have the folder called `ADRESSE` and  files `ADRESSE.*` as described in the instruction. It does have some shape files but none of them seems the correct one. 
Screenshot 2023-02-03 at 15 33 07
```
❯ tree -AC /Users/yuhao.sun/Downloads/BDTOPO_3-3_TOUSTHEMES_SHP_LAMB93_R11_2022-12-15/BDTOPO/1_DONNEES_LIVRAISON_2022-12-00162/BDT_3-3_SHP_LAMB93_R11-ED2022-12-15
/Users/yuhao.sun/Downloads/BDTOPO_3-3_TOUSTHEMES_SHP_LAMB93_R11_2022-12-15/BDTOPO/1_DONNEES_LIVRAISON_2022-12-00162/BDT_3-3_SHP_LAMB93_R11-ED2022-12-15
├── ADMINISTRATIF
│   ├── ARRONDISSEMENT.cpg
│   ├── ARRONDISSEMENT.dbf
│   ├── ARRONDISSEMENT.prj
│   ├── ARRONDISSEMENT.shp
│   ├── ARRONDISSEMENT.shx
│   ├── ARRONDISSEMENT_MUNICIPAL.cpg
│   ├── ARRONDISSEMENT_MUNICIPAL.dbf
│   ├── ARRONDISSEMENT_MUNICIPAL.prj
│   ├── ARRONDISSEMENT_MUNICIPAL.shp
│   ├── ARRONDISSEMENT_MUNICIPAL.shx
│   ├── COLLECTIVITE_TERRITORIALE.cpg
│   ├── COLLECTIVITE_TERRITORIALE.dbf
│   ├── COLLECTIVITE_TERRITORIALE.prj
│   ├── COLLECTIVITE_TERRITORIALE.shp
│   ├── COLLECTIVITE_TERRITORIALE.shx
│   ├── COMMUNE.cpg
│   ├── COMMUNE.dbf
│   ├── COMMUNE.prj
│   ├── COMMUNE.shp
│   ├── COMMUNE.shx
│   ├── COMMUNE_ASSOCIEE_OU_DELEGUEE.cpg
│   ├── COMMUNE_ASSOCIEE_OU_DELEGUEE.dbf
│   ├── COMMUNE_ASSOCIEE_OU_DELEGUEE.prj
│   ├── COMMUNE_ASSOCIEE_OU_DELEGUEE.shp
│   ├── COMMUNE_ASSOCIEE_OU_DELEGUEE.shx
│   ├── DEPARTEMENT.cpg
│   ├── DEPARTEMENT.dbf
│   ├── DEPARTEMENT.prj
│   ├── DEPARTEMENT.shp
│   ├── DEPARTEMENT.shx
│   ├── EPCI.cpg
│   ├── EPCI.dbf
│   ├── EPCI.prj
│   ├── EPCI.shp
│   ├── EPCI.shx
│   ├── REGION.cpg
│   ├── REGION.dbf
│   ├── REGION.prj
│   ├── REGION.shp
│   └── REGION.shx
├── ADMINISTRATIF.md5
├── BATI
│   ├── BATIMENT.cpg
│   ├── BATIMENT.dbf
│   ├── BATIMENT.prj
│   ├── BATIMENT.shp
│   ├── BATIMENT.shx
│   ├── CIMETIERE.cpg
│   ├── CIMETIERE.dbf
│   ├── CIMETIERE.prj
│   ├── CIMETIERE.shp
│   ├── CIMETIERE.shx
│   ├── CONSTRUCTION_LINEAIRE.cpg
│   ├── CONSTRUCTION_LINEAIRE.dbf
│   ├── CONSTRUCTION_LINEAIRE.prj
│   ├── CONSTRUCTION_LINEAIRE.shp
│   ├── CONSTRUCTION_LINEAIRE.shx
│   ├── CONSTRUCTION_PONCTUELLE.cpg
│   ├── CONSTRUCTION_PONCTUELLE.dbf
│   ├── CONSTRUCTION_PONCTUELLE.prj
│   ├── CONSTRUCTION_PONCTUELLE.shp
│   ├── CONSTRUCTION_PONCTUELLE.shx
│   ├── CONSTRUCTION_SURFACIQUE.cpg
│   ├── CONSTRUCTION_SURFACIQUE.dbf
│   ├── CONSTRUCTION_SURFACIQUE.prj
│   ├── CONSTRUCTION_SURFACIQUE.shp
│   ├── CONSTRUCTION_SURFACIQUE.shx
│   ├── LIGNE_OROGRAPHIQUE.cpg
│   ├── LIGNE_OROGRAPHIQUE.dbf
│   ├── LIGNE_OROGRAPHIQUE.prj
│   ├── LIGNE_OROGRAPHIQUE.shp
│   ├── LIGNE_OROGRAPHIQUE.shx
│   ├── PYLONE.cpg
│   ├── PYLONE.dbf
│   ├── PYLONE.prj
│   ├── PYLONE.shp
│   ├── PYLONE.shx
│   ├── RESERVOIR.cpg
│   ├── RESERVOIR.dbf
│   ├── RESERVOIR.prj
│   ├── RESERVOIR.shp
│   ├── RESERVOIR.shx
│   ├── TERRAIN_DE_SPORT.cpg
│   ├── TERRAIN_DE_SPORT.dbf
│   ├── TERRAIN_DE_SPORT.prj
│   ├── TERRAIN_DE_SPORT.shp
│   ├── TERRAIN_DE_SPORT.shx
│   ├── TOPONYMIE_BATI.cpg
│   ├── TOPONYMIE_BATI.dbf
│   ├── TOPONYMIE_BATI.prj
│   ├── TOPONYMIE_BATI.shp
│   └── TOPONYMIE_BATI.shx
├── BATI.md5
├── HYDROGRAPHIE
│   ├── BASSIN_VERSANT_TOPOGRAPHIQUE.cpg
│   ├── BASSIN_VERSANT_TOPOGRAPHIQUE.dbf
│   ├── BASSIN_VERSANT_TOPOGRAPHIQUE.prj
│   ├── BASSIN_VERSANT_TOPOGRAPHIQUE.shp
│   ├── BASSIN_VERSANT_TOPOGRAPHIQUE.shx
│   ├── COURS_D_EAU.cpg
│   ├── COURS_D_EAU.dbf
│   ├── COURS_D_EAU.prj
│   ├── COURS_D_EAU.shp
│   ├── COURS_D_EAU.shx
│   ├── DETAIL_HYDROGRAPHIQUE.cpg
│   ├── DETAIL_HYDROGRAPHIQUE.dbf
│   ├── DETAIL_HYDROGRAPHIQUE.prj
│   ├── DETAIL_HYDROGRAPHIQUE.shp
│   ├── DETAIL_HYDROGRAPHIQUE.shx
│   ├── NOEUD_HYDROGRAPHIQUE.cpg
│   ├── NOEUD_HYDROGRAPHIQUE.dbf
│   ├── NOEUD_HYDROGRAPHIQUE.prj
│   ├── NOEUD_HYDROGRAPHIQUE.shp
│   ├── NOEUD_HYDROGRAPHIQUE.shx
│   ├── PLAN_D_EAU.cpg
│   ├── PLAN_D_EAU.dbf
│   ├── PLAN_D_EAU.prj
│   ├── PLAN_D_EAU.shp
│   ├── PLAN_D_EAU.shx
│   ├── SURFACE_HYDROGRAPHIQUE.cpg
│   ├── SURFACE_HYDROGRAPHIQUE.dbf
│   ├── SURFACE_HYDROGRAPHIQUE.prj
│   ├── SURFACE_HYDROGRAPHIQUE.shp
│   ├── SURFACE_HYDROGRAPHIQUE.shx
│   ├── TOPONYMIE_HYDROGRAPHIE.cpg
│   ├── TOPONYMIE_HYDROGRAPHIE.dbf
│   ├── TOPONYMIE_HYDROGRAPHIE.prj
│   ├── TOPONYMIE_HYDROGRAPHIE.shp
│   ├── TOPONYMIE_HYDROGRAPHIE.shx
│   ├── TRONCON_HYDROGRAPHIQUE.cpg
│   ├── TRONCON_HYDROGRAPHIQUE.dbf
│   ├── TRONCON_HYDROGRAPHIQUE.prj
│   ├── TRONCON_HYDROGRAPHIQUE.shp
│   └── TRONCON_HYDROGRAPHIQUE.shx
├── HYDROGRAPHIE.md5
├── LIEUX_NOMMES
│   ├── DETAIL_OROGRAPHIQUE.cpg
│   ├── DETAIL_OROGRAPHIQUE.dbf
│   ├── DETAIL_OROGRAPHIQUE.prj
│   ├── DETAIL_OROGRAPHIQUE.shp
│   ├── DETAIL_OROGRAPHIQUE.shx
│   ├── LIEU_DIT_NON_HABITE.cpg
│   ├── LIEU_DIT_NON_HABITE.dbf
│   ├── LIEU_DIT_NON_HABITE.prj
│   ├── LIEU_DIT_NON_HABITE.shp
│   ├── LIEU_DIT_NON_HABITE.shx
│   ├── TOPONYMIE_LIEUX_NOMMES.cpg
│   ├── TOPONYMIE_LIEUX_NOMMES.dbf
│   ├── TOPONYMIE_LIEUX_NOMMES.prj
│   ├── TOPONYMIE_LIEUX_NOMMES.shp
│   ├── TOPONYMIE_LIEUX_NOMMES.shx
│   ├── ZONE_D_HABITATION.cpg
│   ├── ZONE_D_HABITATION.dbf
│   ├── ZONE_D_HABITATION.prj
│   ├── ZONE_D_HABITATION.shp
│   └── ZONE_D_HABITATION.shx
├── LIEUX_NOMMES.md5
├── OCCUPATION_DU_SOL
│   ├── HAIE.cpg
│   ├── HAIE.dbf
│   ├── HAIE.prj
│   ├── HAIE.shp
│   ├── HAIE.shx
│   ├── ZONE_DE_VEGETATION.cpg
│   ├── ZONE_DE_VEGETATION.dbf
│   ├── ZONE_DE_VEGETATION.prj
│   ├── ZONE_DE_VEGETATION.shp
│   └── ZONE_DE_VEGETATION.shx
├── OCCUPATION_DU_SOL.md5
├── SERVICES_ET_ACTIVITES
│   ├── CANALISATION.cpg
│   ├── CANALISATION.dbf
│   ├── CANALISATION.prj
│   ├── CANALISATION.shp
│   ├── CANALISATION.shx
│   ├── ERP.cpg
│   ├── ERP.dbf
│   ├── ERP.prj
│   ├── ERP.shp
│   ├── ERP.shx
│   ├── LIGNE_ELECTRIQUE.cpg
│   ├── LIGNE_ELECTRIQUE.dbf
│   ├── LIGNE_ELECTRIQUE.prj
│   ├── LIGNE_ELECTRIQUE.shp
│   ├── LIGNE_ELECTRIQUE.shx
│   ├── POSTE_DE_TRANSFORMATION.cpg
│   ├── POSTE_DE_TRANSFORMATION.dbf
│   ├── POSTE_DE_TRANSFORMATION.prj
│   ├── POSTE_DE_TRANSFORMATION.shp
│   ├── POSTE_DE_TRANSFORMATION.shx
│   ├── TOPONYMIE_SERVICES_ET_ACTIVITES.cpg
│   ├── TOPONYMIE_SERVICES_ET_ACTIVITES.dbf
│   ├── TOPONYMIE_SERVICES_ET_ACTIVITES.prj
│   ├── TOPONYMIE_SERVICES_ET_ACTIVITES.shp
│   ├── TOPONYMIE_SERVICES_ET_ACTIVITES.shx
│   ├── ZONE_D_ACTIVITE_OU_D_INTERET.cpg
│   ├── ZONE_D_ACTIVITE_OU_D_INTERET.dbf
│   ├── ZONE_D_ACTIVITE_OU_D_INTERET.prj
│   ├── ZONE_D_ACTIVITE_OU_D_INTERET.shp
│   └── ZONE_D_ACTIVITE_OU_D_INTERET.shx
├── SERVICES_ET_ACTIVITES.md5
├── TRANSPORT
│   ├── AERODROME.cpg
│   ├── AERODROME.dbf
│   ├── AERODROME.prj
│   ├── AERODROME.shp
│   ├── AERODROME.shx
│   ├── EQUIPEMENT_DE_TRANSPORT.cpg
│   ├── EQUIPEMENT_DE_TRANSPORT.dbf
│   ├── EQUIPEMENT_DE_TRANSPORT.prj
│   ├── EQUIPEMENT_DE_TRANSPORT.shp
│   ├── EQUIPEMENT_DE_TRANSPORT.shx
│   ├── ITINERAIRE_AUTRE.cpg
│   ├── ITINERAIRE_AUTRE.dbf
│   ├── ITINERAIRE_AUTRE.prj
│   ├── ITINERAIRE_AUTRE.shp
│   ├── ITINERAIRE_AUTRE.shx
│   ├── NON_COMMUNICATION.cpg
│   ├── NON_COMMUNICATION.dbf
│   ├── NON_COMMUNICATION.prj
│   ├── NON_COMMUNICATION.shp
│   ├── NON_COMMUNICATION.shx
│   ├── PISTE_D_AERODROME.cpg
│   ├── PISTE_D_AERODROME.dbf
│   ├── PISTE_D_AERODROME.prj
│   ├── PISTE_D_AERODROME.shp
│   ├── PISTE_D_AERODROME.shx
│   ├── POINT_DE_REPERE.cpg
│   ├── POINT_DE_REPERE.dbf
│   ├── POINT_DE_REPERE.prj
│   ├── POINT_DE_REPERE.shp
│   ├── POINT_DE_REPERE.shx
│   ├── POINT_DU_RESEAU.cpg
│   ├── POINT_DU_RESEAU.dbf
│   ├── POINT_DU_RESEAU.prj
│   ├── POINT_DU_RESEAU.shp
│   ├── POINT_DU_RESEAU.shx
│   ├── ROUTE_NUMEROTEE_OU_NOMMEE.cpg
│   ├── ROUTE_NUMEROTEE_OU_NOMMEE.dbf
│   ├── ROUTE_NUMEROTEE_OU_NOMMEE.prj
│   ├── ROUTE_NUMEROTEE_OU_NOMMEE.shp
│   ├── ROUTE_NUMEROTEE_OU_NOMMEE.shx
│   ├── SECTION_DE_POINTS_DE_REPERE.cpg
│   ├── SECTION_DE_POINTS_DE_REPERE.dbf
│   ├── SECTION_DE_POINTS_DE_REPERE.prj
│   ├── SECTION_DE_POINTS_DE_REPERE.shp
│   ├── SECTION_DE_POINTS_DE_REPERE.shx
│   ├── TOPONYMIE_TRANSPORT.cpg
│   ├── TOPONYMIE_TRANSPORT.dbf
│   ├── TOPONYMIE_TRANSPORT.prj
│   ├── TOPONYMIE_TRANSPORT.shp
│   ├── TOPONYMIE_TRANSPORT.shx
│   ├── TRANSPORT_PAR_CABLE.cpg
│   ├── TRANSPORT_PAR_CABLE.dbf
│   ├── TRANSPORT_PAR_CABLE.prj
│   ├── TRANSPORT_PAR_CABLE.shp
│   ├── TRANSPORT_PAR_CABLE.shx
│   ├── TRONCON_DE_ROUTE.cpg
│   ├── TRONCON_DE_ROUTE.dbf
│   ├── TRONCON_DE_ROUTE.prj
│   ├── TRONCON_DE_ROUTE.shp
│   ├── TRONCON_DE_ROUTE.shx
│   ├── TRONCON_DE_VOIE_FERREE.cpg
│   ├── TRONCON_DE_VOIE_FERREE.dbf
│   ├── TRONCON_DE_VOIE_FERREE.prj
│   ├── TRONCON_DE_VOIE_FERREE.shp
│   ├── TRONCON_DE_VOIE_FERREE.shx
│   ├── VOIE_FERREE_NOMMEE.cpg
│   ├── VOIE_FERREE_NOMMEE.dbf
│   ├── VOIE_FERREE_NOMMEE.prj
│   ├── VOIE_FERREE_NOMMEE.shp
│   ├── VOIE_FERREE_NOMMEE.shx
│   ├── VOIE_NOMMEE.cpg
│   ├── VOIE_NOMMEE.dbf
│   ├── VOIE_NOMMEE.prj
│   ├── VOIE_NOMMEE.shp
│   └── VOIE_NOMMEE.shx
├── TRANSPORT.md5
├── ZONES_REGLEMENTEES
│   ├── FORET_PUBLIQUE.cpg
│   ├── FORET_PUBLIQUE.dbf
│   ├── FORET_PUBLIQUE.prj
│   ├── FORET_PUBLIQUE.shp
│   ├── FORET_PUBLIQUE.shx
│   ├── PARC_OU_RESERVE.cpg
│   ├── PARC_OU_RESERVE.dbf
│   ├── PARC_OU_RESERVE.prj
│   ├── PARC_OU_RESERVE.shp
│   ├── PARC_OU_RESERVE.shx
│   ├── TOPONYMIE_ZONES_REGLEMENTEES.cpg
│   ├── TOPONYMIE_ZONES_REGLEMENTEES.dbf
│   ├── TOPONYMIE_ZONES_REGLEMENTEES.prj
│   ├── TOPONYMIE_ZONES_REGLEMENTEES.shp
│   └── TOPONYMIE_ZONES_REGLEMENTEES.shx
└── ZONES_REGLEMENTEES.md5

9 directories, 288 files
```

### PR DESCRIPTION
**Pull request by sebhoerl**
- Making use of INSEE RP 2019
- Making use of BPE 2021
- Making use of Filosofi 2019 income data (Attention: Covid affected)
- Making use of 2021 IRIS
- Making use of BAN Address database to define home locations
- Removal of outdated BD-TOPO data sets for addresses
- Use of new georeferenced version of SIRENE provided by INSEE

**Fixes**
- Fixes #151. 
- Fixes #112. 
- Fixes #94.

**TODO**
- [x] Cleanup some introduced changes (e.g. Monte Carlo output)
- [x] Update documentation with new data sets
- [x] Some minor things (see review)
- [x] **Update unit tests**

### CODE REVIEW

File: data/sirene/localized.py [hunk 1]
```diff
@@ -1,175 +1,33 @@
...
-    df_fixed = df_fixed[~f_missing].copy()
-    df_fixed["status"] = 2
-
-    # Merge data sets
-    df_valid = pd.concat([df_valid, df_fixed])
-    del df_fixed
+    df_siret_geoloc = context.stage("data.sirene.raw_geoloc")
 
-    df_valid = gpd.GeoDataFrame(df_valid, crs = "EPSG:2154")
+    # rassemblement du fichier localisé avec celui siren déjà nettoyé
```
Comment (sebhoerl) [inline data/sirene/localized.py:18]: Translate :)

File: documentation/info/collect.py [hunk 2]
```diff
@@ -71,27 +69,30 @@ def execute(context):
...
-        "final_count": context.get_info("data.bdtopo.cleaned", "final_count"),
-    }
-
+# # 
+#     BDTOPO
+#     info["bdtopo"] = {
+#         "initial_count": context.get_info("data.bdtopo.cleaned", "initial_count"),
+#         "final_count": context.get_info("data.bdtopo.cleaned", "final_count"),
+#     }
+
```
Comment (sebhoerl) [inline documentation/info/collect.py:84]: This is deprecated now, we can just remove it

File: documentation/info/collect.py [hunk 3]
```diff
@@ -71,27 +69,30 @@ def execute(context):
...
+    # info["sirene"] = {
+    #     # "initial_count": context.get_info("data.sirene.localized", "initial_count"),
+    #     # "exact_count": context.get_info("data.sirene.localized", "exact_count"),
+    #     "no_municipality_count": context.get_info("data.sirene.localized", "no_municipality_count"),
+    #     "levenshtein_count": context.get_info("data.sirene.localized", "levenshtein_count"),
+    # }
+
+    # for key, value in info.items():
+    #     print(key, '->', value)
+    # exit()
```
Comment (sebhoerl) [inline documentation/info/collect.py:95]: Same here

File: documentation/info/tex.py [hunk 4]
```diff
@@ -25,25 +23,25 @@ def execute(context):
...
+        # "infoMatchedSirene": "{:,.0f}".format(
+        #     info["sirene"]["exact_count"] +
+        #     info["sirene"]["no_municipality_count"] +
+        #     info["sirene"]["levenshtein_count"]
+        # ),
+        # "infoMatchedSireneRelative": "{:.2f}\\%".format(100 * (
+        #     info["sirene"]["exact_count"] +
+        #     info["sirene"]["no_municipality_count"] +
+        #     info["sirene"]["levenshtein_count"]
+        # ) / info["sirene"]["initial_count"]),
```
Comment (sebhoerl) [inline documentation/info/tex.py:44]: Deprecated stuff can be remoevd

File: documentation/paper.py [hunk 5]
```diff
@@ -3,7 +3,7 @@
 def configure(context):
     context.stage("documentation.plots.data.hts_comparison")
     #context.stage("documentation.plots.theory.sampling_error")
-    context.stage("documentation.plots.monte_carlo")
+    # context.stage("documentation.plots.monte_carlo")
```
Comment (sebhoerl) [inline documentation/paper.py:6]: Activate again, actually, can we maybe add a config switch that activates this or not? Then we can generate the paper plots without during the MC experiment

File: documentation/paper.py [hunk 6]
```diff
@@ -32,8 +32,8 @@ def execute(context):
 
         #"sampling_sample_count.pdf": ("documentation.plots.sampling.sample_count", "sample_count.pdf"),
         #"sampling_error_probability.pdf": ("documentation.plots.sampling.error_probability", "error_probability.pdf"),
-        "monte_carlo.pdf": ("documentation.plots.monte_carlo", "monte_carlo.pdf"),
-        "monte_carlo_table.tex": ("documentation.plots.monte_carlo", "monte_carlo_table.tex"),
+        # "monte_carlo.pdf": ("documentation.plots.monte_carlo", "monte_carlo.pdf"),
+        # "monte_carlo_table.tex": ("documentation.plots.monte_carlo", "monte_carlo_table.tex"),
```
Comment (sebhoerl) [inline documentation/paper.py:36]: Again, but maybe with switch (see above)

File: data/ban/raw.py [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+            #concaténation des départements      
+            df_ban = pd.concat([df_ban,df_ban_tmp])
+
+            progress.update()
+            
+            
+
+    #conversion en geo dataframe projeté en Lambert
+    df_ban = gpd.GeoDataFrame(df_ban, geometry=gpd.points_from_xy(df_ban.x, df_ban.y),crs=2154)
```
[the remark this answers is not in the document]
Comment (aburianne) [reply data/ban/raw.py:49]: https://pyproj4.github.io/pyproj/stable/api/crs/crs.html#pyproj.crs.CRS.from_user_input
every syntax is okay, but to be constant i set this everywhere : "EPSG:2154"

File: data/bdtopo/raw.py [hunk 8]
```diff
@@ -10,34 +10,33 @@
...
-                if code[:2] in requested_departements or code[:3] in requested_departements:
+                # we check for every buildings if it has residential use (only or mix use)
+                if item['properties']["NB_LOGTS"] is not None and int(item['properties']["NB_LOGTS"]) > 0:
+                   
                     df_bdtopo.append(dict(
-                        geometry = geo.Point(*item["geometry"]["coordinates"]),
-                        commune_id = item["properties"]["CODE_INSEE"],
-                        raw_number = pd.to_numeric(item["properties"]["NUMERO"]),
-                        raw_street = item["properties"]["NOM_1"]
+                        geometry = geo.Point(*item["geometry"]["coordinates"][0][0][0:2]),
```
Comment (sebhoerl) [inline data/bdtopo/raw.py:31]: I assume that these are polygons? Then you could just do
`item["geometry"].centroid`

File: data/bdtopo/raw.py [hunk 9]
```diff
@@ -10,34 +10,33 @@
...
+                        bdtopo_id = item["properties"]["ID"]
                     ))
 
                 progress.update()
 
     df_bdtopo = pd.DataFrame.from_records(df_bdtopo)
     df_bdtopo = gpd.GeoDataFrame(df_bdtopo, crs = "EPSG:2154")
-
+    df_bdtopo = df_bdtopo[df_bdtopo.is_valid].copy()
```
Comment (sebhoerl) [inline data/bdtopo/raw.py:39]: What does this do?

File: data/bdtopo/raw.py (hunk 8 above)
Comment (aburianne) [reply data/bdtopo/raw.py]: makes sens, done

File: tests/test_pipeline.py [hunk 10]
```diff
@@ -33,6 +33,7 @@ def test_data(tmpdir):
     assert os.path.isfile("%s/ile_de_france_hts_persons.csv" % output_path)
     assert os.path.isfile("%s/ile_de_france_hts_trips.csv" % output_path)
     assert os.path.isfile("%s/ile_de_france_sirene.gpkg" % output_path)
+    # ban?
```
Comment (sebhoerl) [inline tests/test_pipeline.py:36]: Remove

File: docs/population.md [hunk 11]
```diff
@@ -91,85 +91,100 @@ guarantee that you have exactly the correct format), you should make sure that
...
   - `BDTOPO`
-  - `1_DONNEES_LIVRAISON_2021-01-00120`
-  - `BDT_3-0_SHP_LAMB93_R11-ED2020-12-15`
-  - `ADRESSES`
-- Copy the files `ADRESSE.*` from the folder `ADDRESSES` in *shape file* format into `data/bdtopo`.
+  - `1_DONNEES_LIVRAISON_some_date`
+  - `BDT_3-0_SHP_LAMB93_R11-some_date`
+  - `BATI`
+- Copy the files `BATIMENT.*` from the folder `BATI` in *shape file* format into `data/bdtopo`.
+
```
[the remark this answers is not in the document]
Comment (aburianne) [reply docs/population.md]: yep, it's all kind of builidings, you are right about this 👍

**Pull request merged**